x-pack/plugins/secops/public/components/timeline/body/renderers/zeek_signature.tsx

```diff
@@ -0,0 +1,234 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { EuiBadge, EuiFlexGroup, EuiFlexItem } from '@elastic/eui';
+import { get } from 'lodash/fp';
+import * as React from 'react';
+import { pure } from 'recompose';
+import styled from 'styled-components';
+
+import { Ecs } from '../../../../graphql/types';
+import { DragEffects, DraggableWrapper } from '../../../drag_and_drop/draggable_wrapper';
+import { escapeDataProviderId } from '../../../drag_and_drop/helpers';
+import { Provider } from '../../../timeline/data_providers/provider';
+
+import { GoogleLink } from './google_link';
+import { TotalVirusLink } from './total_virus_link';
+
+import * as i18n from './translations';
+
+const Badge = styled(EuiBadge)`
+  vertical-align: top;
+`;
+
+const TokensFlexItem = styled(EuiFlexItem)`
+  margin-left: 3px;
+`;
+
+type StringRenderer = (value: string) => string;
+
+export const defaultStringRenderer: StringRenderer = (value: string) => value;
+
+export const moduleStringRenderer: StringRenderer = (value: string) =>
+  value.split('.')[1] != null ? value.split('.')[1] : value;
```

Comment:
This is a most-likely a non-issue, but basic strings with a single trailing `.` will return `emptystring` instead what's before it.

``` javascript
test('renders modules of bad input with multiple dots as only the second value', () => {
  expect(moduleStringRenderer('abc.')).toEqual('abc');
});
```

Reply:
I like the behavior. I'll add that test though to be explicit about what to expect with bad input in case we need/want to change that down the road.

Reply:
Re-wrote it a bit and then added the tests so if it hits that edge case it will work out well.